deployments/linea-goerli/usdc/relations.ts

```diff
@@ -0,0 +1,34 @@
+import baseRelationConfig from '../../relations';
+
+export default {
+  ...baseRelationConfig,
+  governor: {
+    artifact: 'contracts/bridges/linea/LineaBridgeReceiver.sol:LineaBridgeReceiver'
+  },
+  ClonableBeaconProxy: {
+    artifact: 'contracts/ERC20.sol:ERC20'
+  },
```

Comment:
Is this needed? Seems specific to Arbitrum

Reply:
You're right, I remove it.